docs/ramp_fitting/source/main.rst

```diff
@@ -37,7 +37,11 @@ a given pixel, the final slope is determined as a weighted average from all
 intervals and is written to a file as the primary output product.  In this
 output product, the 4-D GROUPDQ from all integrations is compressed into 2-D,
 which is then merged (using a bitwise OR) with the input 2-D PIXELDQ to create
-the output PIXELDQ.
+the output PIXELDQ.  If the ramp fitting step is run by itself, the output file 
+name will have the suffix '_RampFit'; if the ramp fitting step is run as the 
```

Comment:
Here's the first line from each of our default step and pipeline `*.cfg` files:
```
name = "alignrefs"
name = "ami_analyze"
name = "ami_average"
name = "ami_normalize"
name = "assign_wcs"
name = "Ami3Pipeline"
name = "Coron3Pipeline"
name = "DarkPipeline"
name = "Image2Pipeline"
name = "Image3Pipeline"
name = "SloperPipeline"
name = "Spec2Pipeline"
name = "combine_1d"
name = "cube_build"
name = "dark_current" 
name = "dq_init"
name = "emission" 
name = "engdblog"
name = "extract_1d"
name = "extract_2d" 
name = "flat_field" 
name = "fringe" 
name = "hlsp"
name = "imprint" 
name = "ipc" 
name = "jump"
name = "klip"
name = "lastframe" 
name = "linearity"
name = "outlier_detection" 
name = "pathloss" 
name = "Persistence"
name = "photom"
name = "RampFit"
name = "refpix"
name = "resample"
name = "reset" 
name = "rscd" 
name = "saturation"
name = "skymatch"
name = "source_catalog"
name = "srctype" 
name = "stackrefs"
name = "straylight"
name = "subtract_images"
name = "superbias" 
name = "tweakreg"
name = "tweakreg_catalog"
name = "WfsCombine"
```

Agree that it would be good to make them consistent (and lowercase?) at some point.